Configuration/StandardSequences/python/ValidationCosmics_cff.py

```diff
@@ -10,8 +10,8 @@
 
 from Validation.Configuration.globalValidationCosmics_cff import *
 
-prevalidation = cms.Sequence(globalPrevalidationCosmics)
+prevalidationCosmics = cms.Sequence(globalPrevalidationCosmics)
```

Comment:
hello @abbiendi @HuguesBrun 
what is the reason why you rename these two sequences,
given that you don't use them anywhere else in your PR?

I've checked in various test.py scattered are around 81X (outside the central sequences exercised by the short matrix)
and nowhere could I find where the prevalidation which is defined in ValidationCosmics_cff.py  is used

As such, this PR could go... if I haven't missed anything!